docs/using-est-with-certd.md

```diff
@@ -0,0 +1,187 @@
+# Using EST with Certificates Service
+
+This document covers how to configure Certificates Service to issue certificates over EST.
```

Comment:
This would also be a good place to re-emphasize the purpose of this doc and also when to NOT read this document (e.g. This document addresses the more advanced scenario of configuring the Certificates Service to issue arbitrary certificates over EST.  Refer to the [other doc] that covers the common scenario of issuing the device ID and Edge CA certs.)

Reply:
Yes, I agree.. in fact we need to put a link to the other doc that talks about IoT Edge scenarios..

Reply:
Updated